### ISSUE TITLE
can we filter based on company staff?

### ISSUE THREAD

**Issue by zkoppert** [github/contributors#50]
This would be useful for GItHub and potentially for other companies too.

**Comment by zkoppert** [comment]
Similar GitHub request, 

> feature request, could we have a column for "non-org member" so we can see potentially who is a Hubber vs. not easily

**Comment by hectorsector** [comment]
My team has a similar need, and we would actually want to get more granular and filter by team membership. We work in a large team, and we often want to see the effects of our work on the community, but also on other Hubbers that aren't a part of our team.

**Comment by jmeridth** [comment]
> My team has a similar need, and we would actually want to get more granular and filter by team membership. We work in a large team, and we often want to see the effects of our work on the community, but also on other Hubbers that aren't a part of our team.

@hectorsector to confirm, you're looking to filter by one or more team names?  If we filter on multiple we'd want the team name in the output, correct?  Do we have any security concerns of sharing a user's team name?  Typically team names are not exposed publicly (I may have to triple check this).  I assume that is to avoid spam.

**Comment by hectorsector** [comment]
> you're looking to filter by one or more team names? If we filter on multiple we'd want the team name in the output

That's correct. Team name would contain org information as well so something like `@github/docs-localization`.

> Do we have any security concerns of sharing a user's team name? Typically team names are not exposed publicly (I may have to triple check this). I assume that is to avoid spam.

Team names are already used in public issues when at-mentioning that team, although [visibility](https://docs.github.com/en/organizations/organizing-members-into-teams/about-teams#team-visibility) of the team members is restricted.

For our specific use case, we don't need the team name published as long as we can filter by that team information, if that helps.
